docs/_docs/tech/ui-components.md

```diff
@@ -0,0 +1,46 @@
+---
+title: UI components
+permalink: /docs/ui-components
+---
+
+# UI components
+
+For use in your own application, Weaverbird exports a bunch of Vue components:
+
+## Vqb: the all-in-one component
+
+The `Vqb` component contains the full UI of Weaverbird.
+
+To run correctly, it assumes the Vqb Vuex store module is present. The helper `registerModule` take care of this part: provide it its initial state and the translator you wish to use. 
+
+```js
+import { registerModule, Vqb } from 'weaverbird';
+
+new Vue({
+  el: '#app',
+  components: {
+    Vqb,
+  },
+  created: function() {
+    registerModule(this.$store, {
+      currentPipelineName: 'pipeline',
+      pipelines: {
+        pipeline: [ ...]
+      },
+      translator: 'mongo40',
+    });
+  }
+});
+
+```
+
+> An example of such integration is provided in `playground/dist/app.js`.
+
+## Main sub-components
+
+The `Vqb` component consists is three sub-components, that you can use as well to fit it nicely into your app.
+
+- `QueryBuilder` is the main left panel, where user see the current pipeline and can edit its steps.
+- `DataViewer` is the main right panel, where the data table resides and the associated actions.
+- `PipelineSelector` is the simple dropdown to select the currently displayed pipeline.
+
```

Comment:
It misses a word about the `FilterEditor` (https://github.com/ToucanToco/weaverbird/blob/master/src/main.ts#L20)
```
`FilterEditor` is needed for internal purpose in ToucanToco. It must not be used has its api is uncertain.
```

Reply:
Also, this message could be added in a docstring in the `main.ts` abobe https://github.com/ToucanToco/weaverbird/blob/master/src/main.ts#L20 ?

Reply:
I know :) I already prepared this, but as this PR may be merged before #494, I think it's better to add it in the PR that adds the component:

```
## Utility components

Weaverbird also exports some of its internal components so they can be reused elsewhere:

- `FilterEditor` is the main component of the Filter step. It can display and edit trees of filtering conditions.
```

Reply:
Adding a doc string to separate clearly the components in main.ts is a great idea, doing thatright now

Reply:
Reverting what I said, you're right it's already exported :joy: 
So I'm adding the doc abut FilerEditor in this PR

Thanks for noticing!

Reply:
I really think we should specify that these "utility components" (only the `FilterEditor` for now) `are for Toucan Toco internal purpose`. Maybe, we will decide in the futur that they are part of the weaverbird api, but maybe we will find an other way. It is too soon to be sure !

Reply:
Not convinced there, it's clear the API as we've not reach the 1.0 version.
I can add a message though indicating that these components are not directly part of the external API and changes can happen on them.